weave/index/index_sqlite.py

```diff
@@ -99,60 +100,11 @@ def generate_index(self, **kwargs):
         basket_jsons = _get_list_of_basket_jsons(self.pantry_path,
                                                  self.file_system)
 
-        storage_type = self.file_system.__class__.__name__
-
-        index_columns = get_index_column_names()
-        num_index_columns = len(index_columns)
-        index_columns = ", ".join(index_columns)
-
-        bad_baskets = []
         for basket_json_address in basket_jsons:
-            with self.file_system.open(basket_json_address, "rb") as file:
-                basket_dict = json.load(file)
-
-                # If the basket is invalid, add it to a list, then skip it.
-                if not validate_basket_dict(basket_dict):
-                    bad_baskets.append(os.path.dirname(basket_json_address))
```

Comment:
I didn't know what was wrong here, so I just mirrored how we're doing it in the SQL index and offload most of this logic to the create_index_from_fs function